article/app/controllers/LiveBlogController.scala

```diff
@@ -61,16 +69,25 @@ class LiveBlogController(
           case (minute: MinutePage, HtmlFormat) =>
             Future.successful(common.renderHtml(MinuteHtmlPage.html(minute), minute))
           case (blog: LiveBlogPage, HtmlFormat) => {
-            // dcrCanRender is always false because right now blogs are not supported by DCR
+            // dcrCouldRender is always false because right now blogs are not supported by DCR
             // but we included this variable as an indication of what is going to be possible in the future
-            val dcrCanRender = false
+            val dcrCouldRender = false
             val participatingInTest = ActiveExperiments.isParticipating(LiveblogRendering)
+            val properties =
+              Map(
+                "participatingInTest" -> participatingInTest.toString(),
+                "dcrCouldRender" -> dcrCouldRender.toString(),
+              )
             val remoteRendering =
-              shouldRemoteRender(request.forceDCROff, request.forceDCR, participatingInTest, dcrCanRender)
+              shouldRemoteRender(request.forceDCROff, request.forceDCR, participatingInTest, dcrCouldRender)
 
             remoteRendering match {
```

Comment:
pattern matching on booleans is a bit overkill - might be nicer to just use an if/else